lib/rubygems/specification.rb

```diff
@@ -2591,6 +2591,7 @@ def traverse trail = [], visited = {}, &block
 
   def validate packaging = true
     @warnings = 0
+    require 'rubygems/util/spdx' unless defined?(Gem::SPDX)
```

Comment:
@kemitchell I'm really sorry, I think I pointed you in the wrong direction.  Try changing this to an `autoload` on the constant rather than requiring in the method.  I'm really sorry!